lib/kafka_ex/protocol/metadata.ex

```diff
@@ -93,24 +97,69 @@ defmodule KafkaEx.Protocol.Metadata do
     }
   end
 
-  def create_request(correlation_id, client_id, ""), do: KafkaEx.Protocol.create_request(:metadata, correlation_id, client_id) <> << 0 :: 32-signed >>
+  def create_request(correlation_id, client_id, ""), do: create_request(correlation_id, client_id, "", @default_api_version)
```

Comment:
minor nit: these two function heads could be consolidated by using a default value for `api_version` in `create_request/4`

Reply:
done.